bundle/wildfly/pom.xml

```diff
@@ -0,0 +1,254 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>bundle-parent</artifactId>
+        <groupId>org.bonitasoft.distrib</groupId>
+        <version>7.3.0-SNAPSHOT</version>
+    </parent>
+    <groupId>org.bonitasoft.distrib</groupId>
+    <artifactId>bundle-wildfly</artifactId>
+    <name>wildfly bundle</name>
+    <packaging>pom</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>ant-contrib</groupId>
+            <artifactId>ant-contrib</artifactId>
+        </dependency>
+    </dependencies>
+
+    <properties>
+        <if.task>net.sf.antcontrib.logic.IfTask</if.task>
+
+        <zipName>BonitaBPMCommunity-${project.version}-wildfly-${wildfly.version}</zipName>
+        <zipFolder>${user.home}</zipFolder>
+
+        <wildfly.zip.name>wildfly-${wildfly.version}</wildfly.zip.name>
+        <wildfly.bos.home.dir>${project.build.directory}/${zipName}</wildfly.bos.home.dir>
+        <wildfly.home>${wildfly.bos.home.dir}/${wildfly.zip.name}</wildfly.home>
+        <platform.setup.folder>${wildfly.home}/setup</platform.setup.folder>
+        <wildfly.default.dir>${wildfly.home}/standalone/</wildfly.default.dir>
+        <wildfly.deploy.dir>${wildfly.default.dir}/deployments</wildfly.deploy.dir>
+        <wildfly.conf.dir>${wildfly.default.dir}/configuration</wildfly.conf.dir>
+        <wildfly.lib.dir>${wildfly.default.dir}/lib</wildfly.lib.dir>
+        <wildfly.modules.dir>${wildfly.home}/modules/system/layers/base</wildfly.modules.dir>
+        <spring.module.relative.path>org/springframework/spring/main</spring.module.relative.path>
+        <wildfly.spring.module.dir>${wildfly.modules.dir}/${spring.module.relative.path}</wildfly.spring.module.dir>
+        <wildfly.download.url>http://download.jboss.org/wildfly/${wildfly.version.minor}/${wildfly.zip.name}.zip</wildfly.download.url>
```

Comment:
Wrong url.
should be http://download.jboss.org/wildfly/9.0.2.Final/wildfly-9.0.2.Final.zip
```diff
- http://download.jboss.org/wildfly/${wildfly.version.minor}/${wildfly.zip.name}.zip
+ http://download.jboss.org/wildfly/${wildfly.version}/${wildfly.zip.name}.zip
```